tests/integration/components/polaris-resource-list/item-test.js

```diff
@@ -315,8 +318,31 @@ module('Integration | Component | polaris-resource-list/item', function(hooks) {
           .dom('[data-test-checkbox-input]')
           .hasAttribute('data-test-checkbox-input-checked');
       });
+    }
+  );
 
-      test("renders a disabled checked Checkbox if 'loading' context is true", async function(assert) {
```

Comment:
This test was nested under the `selectMode` module, but in `polaris-react` it looks like it's nested under the `media` module so it hasn't been removed, just moved down into `media` FYI.

Reply:
Yeah looks like I messed up the test layout somehow when I first did these 🤦‍♂️

Reply:
Wait wat... that test has moved since I pulled the React code to port it 😅 Seems like an accident as well, what are your thoughts on keeping it here where it belongs irrespective of what's in the React code?

Reply:
Hm good point. Also that's weird that it moved since you started haha. I'll put it back.